graphene-common/src/main/java/net/iponweb/disthene/reader/utils/DateTimeUtils.java

```diff
@@ -50,4 +50,8 @@ private static long getUnitValue(String s) {
     public static boolean testTimeOffset(String s) {
         return timeOffsetPattern.matcher(s.replaceAll("^['\"]|['\"]$", "")).matches();
     }
+
+    public static long currentTimeSeconds() {
```

Comment:
How about naming currentTimestamp instead?

Reply:
The DateTimeUtils of popular library provides the currentTimeMillis() method. This means that some people are familiar with this context.
So I named the currentTimeSeconds(). Personally I think it's a good pattern to have TimeUnit included in the method name or explicitly get TimeUnit with that value like that `DateTimeUtils.currentTime(TimeUnit.Second)`